src/ploneintranet/activitystream/browser/stream_provider.py

```diff
@@ -90,15 +88,12 @@ def activities(self):
                 i += 1
 
     def _activity_visible(self, activity):
-        if IStatusActivity.providedBy(activity) and self.show_microblog:
+        if IStatusActivity.providedBy(activity):
             return True
         return False
 
     def _activities_statuses(self):
-        if not self.show_microblog:
-            raise StopIteration()
         container = PLONEINTRANET.microblog
```

Comment:
Ok... here I can see this:
 - https://github.com/ploneintranet/ploneintranet/blob/251c8cf9f1e69c38030b6b6ac2f7c93c86ae1e60/src/ploneintranet/core/integration.py#L25

So I think container is always Truish...

If it always evaluate to True, the "if not container" in the line below is redundant.